kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -216,6 +223,26 @@ def exec_foundry_kompile(
             md_selector=md_selector,
             profile=profile,
         )
+    kevm = KEVM(foundry_definition_dir, main_file=foundry_main_file, profile=profile)
+    if regen or rekompile or reparse or not parsed_spec.exists():
+        prove_args = add_include_arg(includes)
+        kevm.prove(
+            foundry_main_file,
+            spec_module_name=spec_module,
+            dry_run=True,
+            args=(['--emit-json-spec', str(parsed_spec)] + prove_args),
+        )
```

Comment:
Ideally we should emit the JSON spec (or the CFGs) directly. Is that feasible?

Reply:
Yes, it's feasible. I thought of it after opening this PR, but figured it could wait.

Reply:
I can change it though.

Reply:
https://github.com/runtimeverification/evm-semantics/issues/1374